samples/Ralltiir/Ralltiir.Server/Ralltiir.Server.csproj

```diff
@@ -0,0 +1,54 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
```

Comment:
For consistency with the other samples, please use 2 spaces instead of 4 for indentation.

Reply:
I added a `.editorconfig` file to hopefully prevent some future issues and then reformatted this file